app/helpers/user_helper.rb

```diff
@@ -0,0 +1,19 @@
+module UserHelper
+
+
+  def has_avatar? user
+    user.avatar.present?
+  end
+
+  def editable_by? user
+    user == current_user
+  end
+
+    def user_path(user)
```

Comment:
You can use the controller version by adding the helper_method declaration in the controller.